framework/core/Utils.pm

```diff
@@ -329,8 +329,9 @@ sub sed_cmd {
     print(STDERR "Executed command: sed -i $cmd_string $file_name\n") if $DEBUG;
 
     # We ignore sed result as it is ok if command fails.
-    if (`uname -s` eq "Darwin" ) {
-        `sed -i '' $cmd_string $file_name`;
+    chomp(my $uname = `uname -s`);
+    if ($uname eq "Darwin" ) {
+        `sed -i '' -e '$cmd_string' $file_name`;
```

Comment:
This seems different than the command that was previously used in test_verify_bugs.sh, which did not set the -e flag. Is this the command?